pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-deny-clusterversion-upgradeconfig/sync.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
```

Comment:
as to sync.yaml, need to move it to somewhere global to all policies and find a way to auto deploy it, we can put it here for now.